src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -377,19 +377,21 @@ export const NavBar: React.FC = track(
                   Shows
                 </NavBarItemLink>
 
+                <NavBarItemLink href="/institutions" onClick={handleClick}>
+                  Museums
+                </NavBarItemLink>
+
                 <NavBarItemLink
-                  // Hide link at smaller viewports — corresponding display inside of `MoreNavMenu`
```

Comment:
This comment no longer applies because the breakpoint for mobile takes over before we'd get to this point.